setup.md

```diff
@@ -0,0 +1,234 @@
+# Using Private Packagist in a Composer project
+
+You can use Private Packagist in a new or in an existing Composer project. Some of the steps in this guide will only apply if your project needs the respective Composer feature explained in that step.
+
+## Creating a new Composer project with Private Packagist
+
+When setting up a project it's important to add both the private repository URL for your Private Packagist organization and to disable packagist.org. Disabling the public open-source repository packagist.org is important because Private Packagist mirrors all the packages you get from there. If you do not disable it, Composer v1 will load all public packages twice which will slow down Composer and use more memory. Composer v2 automatically skips packagist.org packages if they are mirrored through Private Packagist.
```

Comment:
It is not clear from reading this why it still needs to be disable with V2

Reply:
That's kind of the point, it is not needed in v2